test/unit/math/rev/err/check_greater_test.cpp

```diff
@@ -1,10 +1,42 @@
+#include <stan/math/rev/arr.hpp>
 #include <stan/math/rev/mat.hpp>
+#include <stan/math/rev/scal.hpp>
 #include <gtest/gtest.h>
+#include <vector>
 #include <limits>
 
 using stan::math::check_greater;
 using stan::math::var;
 
+TEST(AgradRevErrorHandlingScalar, CheckGreaterVarCheckVectorized) {
+  using stan::math::check_greater;
+  using stan::math::var;
```

Comment:
Already declared outside of test block.

Reply:
I removed the global using and added it inside separate tests.